src/main/java/rayshawn/chatbot/commands/Command.java

```diff
@@ -0,0 +1,70 @@
+package rayshawn.chatbot.commands;
+
+import static rayshawn.chatbot.messages.Messages.LIST_COUNT_MESSAGE;
+
+import java.util.List;
+
+import rayshawn.chatbot.tasks.Task;
+import rayshawn.chatbot.tasks.TaskList;
+
+/**
+ * Represents an executable command.
+ */
+public class Command {
```

Comment:
I love how you created many different classes for each Command. It really improves readability! However, would using enum suffice?